asyncpgsa/connection.py

```diff
@@ -92,18 +92,12 @@ def compile_query(query, dialect=_dialect, inline=False):
 
 
 class SAConnection(connection.Connection):
-    # __slots__ = ('_dialect')
     _dialect = None
 
-    # def __init__(self, connection_: connection, *, dialect=None):
-    #     self._connection = connection_
-    #     if not dialect:
-    #         dialect = _dialect
-    #     self._dialect = dialect
+    def __init__(self, *args, dialect=None, **kwargs):
+        super(SAConnection, self).__init__(*args, **kwargs)
```

Comment:
This can be the new `super().__init(*args, **kwargs)` as there is now reason to support python2